pkg/webhook/yurtappdaemon/v1alpha1/yurtappdaemon_validation.go

```diff
@@ -39,6 +39,10 @@ import (
 	"github.com/openyurtio/openyurt/pkg/apis/apps/v1alpha1"
 )
 
+const (
+	YurtAppDaemonKind = "YurtAppDaemon"
```

Comment:
how about replace all define `"YurtAppDaemon" string in the OpenYurt repo and define `YurtAppDaemonKind` in a proper file?